<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Dialog{
background-color:rgb(245, 245, 245);}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 20pt &quot;Yu Mincho Light&quot;;color:rgb(0,0,0);
font: 25 24pt &quot;Yu Mincho Light&quot;;
font: 25 24pt &quot;Yu Mincho Light&quot;;</string>
   </property>
   <property name="text">
    <string>Data Collection</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>731</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;Yu Mincho Light&quot;;color:rgb(0,0,0);
font: 25 16pt &quot;Yu Mincho Light&quot;;</string>
   </property>
   <property name="text">
    <string>Please type in the initial number of days you want this software to collect data:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="run">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
font: 25 12pt &quot;Yu Mincho Light&quot;;
background-color:rgb(0,0,0);color:rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>RUN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="daysInput">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Segoe Script&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="retry">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0);
font: 25 14pt &quot;Yu Mincho Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stoptime">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;Yu Mincho Light&quot;;color:rgb(0,0,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="instructionOne">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;Yu Mincho Light&quot;;color:rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="instructionTwo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;Yu Mincho Light&quot;;color:rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="next1">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>530</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30px;
background-color:rgb(0,0,0);
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color:rgb(0,0,0);
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
